Makefile

```diff
@@ -1,24 +1,8 @@
-k8s_version ?= 1.14.6
-globalnet ?= false
-deploytool ?= operator
+include Makefile.inc
 
 TARGETS := $(shell ls -p scripts | grep -v -e / -e clusters -e deploy)
```

Comment:
You should remove clusters and deploy from the exclusion since you deleted the scripts

Reply:
Done!